Watchman.Discord/Areas/Responses/Controllers/ResponsesController.cs

```diff
@@ -24,16 +26,20 @@ public class ResponsesController : IController
         private readonly MessagesServiceFactory _messagesServiceFactory;
         private readonly UsersRolesService _usersRolesService;
         private readonly Services.ResponsesService _responsesService;
+        private readonly ResponsesMessageService _responsesMessageService;
+        private readonly string[] possibleArguments;
 
-        public ResponsesController(IQueryBus queryBus, ICommandBus commandBus, UsersService usersService, DirectMessagesService directMessagesService, MessagesServiceFactory messagesServiceFactory, UsersRolesService usersRolesService, Services.ResponsesService responsesService)
+        public ResponsesController(IQueryBus queryBus, ICommandBus commandBus, UsersService usersService, DirectMessagesService directMessagesService, MessagesServiceFactory messagesServiceFactory, UsersRolesService usersRolesService, Services.ResponsesService responsesService, ResponsesMessageService responsesMessageService)
         {
             this._queryBus = queryBus;
             this._commandBus = commandBus;
             this._usersService = usersService;
             this._directMessagesService = directMessagesService;
             this._messagesServiceFactory = messagesServiceFactory;
             this._responsesService = responsesService;
-            _usersRolesService = usersRolesService;
+            this._usersRolesService = usersRolesService;
+            this._responsesMessageService = responsesMessageService;
+            this.possibleArguments = new string[] { "all", "default", "custom" };
```

Comment:
czemu ustawiasz to tutaj? 
wystarczy 
`private readonly string[] possibleArguments = new string[] { "all", "default", "custom" };`